### PR TITLE
[15.0][ADD] repair_reason

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Define repair reasons, such as, "Non conformed product".
Add this reason in the repair order form view

Potentially this can be connected with help desk and RMAs to propagate the reason from there.

cc @ForgeFlow

### CODE REVIEW

File: repair_reason/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 ForgeFlow S. L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Repair Root Cause",
```
Comment (LoisRForgeFlow) [inline repair_reason/__manifest__.py]: Repair Reason to align with technical name repair_reason?

File: repair_reason/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "Repair Reasons",
```
Comment (LoisRForgeFlow) [inline repair_reason/__manifest__.py]: This category does not exist, check the base repair module and use the same category

**Review by LoisRForgeFlow — COMMENTED** [review]
Some small comments

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by OriolMForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**